node/src/neighborhood/mod.rs

```diff
@@ -1754,21 +2253,137 @@ mod tests {
     }
 
     #[test]
-    #[should_panic(
-        expected = "Neighbor masq://eth-ropsten:AQIDBA@1.2.3.4:1234 is not on the mainnet blockchain"
-    )]
-    fn cant_create_mainnet_neighborhood_with_non_mainnet_neighbors() {
-        let cryptde = main_cryptde();
-        let earning_wallet = make_wallet("earning");
-        let mut bc = bc_from_nc_plus(
-            NeighborhoodConfig {
-                mode: NeighborhoodMode::ConsumeOnly(vec![NodeDescriptor::try_from((
-                    cryptde,
-                    "masq://eth-ropsten:AQIDBA@1.2.3.4:1234",
-                ))
-                .unwrap()]),
-                min_hops: MIN_HOPS_FOR_TEST,
-            },
+    fn test_wtih_standard_gossip_have_new_exit_node_in_database() {
+        let mut subject = make_standard_subject();
+        let root_node_key = subject.neighborhood_database.root_key().clone();
+        let root_node = subject.neighborhood_database.root().clone();
+        let first_neighbor = make_node_record(1111, true);
+        let second_neighbor = make_node_record(2222, true);
```

Comment:
Kdyz mluvis o nejakych Nodech takto obecne (ze je nepojmenujes treba podle jejich role jako "introducer", "introducee" apod.), silne doporucuji pouzivat radeji to jeste vic neutralni oznaceni `first_node`, `second_node`. V jistych pripadech by si nekdo mohl vykladat, ze slovem `neighbor` myslis primeho souseda.